core/utxo/utxo.go

```diff
@@ -729,6 +732,9 @@ func (uv *UtxoVM) SelectUtxos(fromAddr string, fromPubKey string, totalNeed *big
 				uv.xlog.Trace("utxo still frozen, skip it", "key", string(key), "fheight", uItem.FrozenHeight)
 				continue
 			}
+			keyTuple := bytes.Split(key[1:], []byte("_")) // [1:] 是为了剔除表名字前缀
+			refTxid, _ := hex.DecodeString(string(keyTuple[len(keyTuple)-2]))
+			offset, _ := strconv.Atoi(string(keyTuple[len(keyTuple)-1]))
```

Comment:
could be a util function for duplicate code?

Reply:
yes, already fix